src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -144,10 +144,14 @@ void GcsPlacementGroupScheduler::Schedule(
                     {key, (*bundle_locations)[iter->BundleId()].first.Binary()});
               }
 
-              // Update `node_to_leased_bundles_`.
               for (const auto &iter : *bundle_locations) {
                 const auto &location = iter.second;
                 node_to_leased_bundles_[location.first].push_back(location.second);
+                placement_group_bundle_locations_[placement_group->GetPlacementGroupID()]
+                                                 [location.second->Index()] =
+                                                     location.first;
+                placement_group->GetMutableBundle(location.second->Index())
```

Comment:
Can we avoid adding the `is_placed` boolean? It seems sufficient that a location is assigned to infer that a bundle is placed. In the GetUnplacedBundles() method we can check if the location is present for the bundle.